staff/giorgio-delabarrera/spotify-app/scripts/main.js

```diff
@@ -0,0 +1,108 @@
+
+function Container(tag) {
+    this.element = document.createElement(tag || 'div');
+}
+
+Container.prototype.empty = function() {
+    this.element.innerHTML = '';
+};
+
+// my presentation logic
+
+logic.token = 'BQDzDhEaitwIB7HjGuoCBcJ1IsCORLKotN9s7td-JMoubvlFcUEdmB6l-HRok2Irf1qCJp8Z6yKH-2hHzNY';
+
+var DEFAULT_COVER_IMAGE = 'https://media2.fishtank.my/app_themes/era/assets/images/default-album-art.png';
+var ERROR_MESSAGE = 'Sorry, we have temporary problem, try again later.';
+
+// components
+var search = new SearchPanel();
+var artistsList = new ResultsList();
+var albumsList = new ResultsList();
+var tracksList = new ResultsList();
+
+// containers
+var listContainer = new Container();
```

Comment:
I'm a bit unclear on this -- does it not make sense to re-use some of the components from "web-components", or is this the most efficient route?

Reply:
I think I could have a Container component and use it. Although this is the case, I only use Container NO as a component but only as an object within this file